.fixtures.yml

```diff
@@ -0,0 +1,8 @@
+fixtures:
+  forge_modules:
+    stdlib: puppetlabs/stdlib
```

Comment:
can you please use git urls instead of puppet forge? in general we try to run unit tests on unreleased changes from dependencies and forge releases for acceptance tests.